src/Symfony/Component/HttpKernel/DataCollector/ConfigDataCollector.php

```diff
@@ -242,4 +258,59 @@ public function getName()
     {
         return 'config';
     }
+
+    /**
+     * Tries to retrieve information about the current Symfony version.
+     *
+     * @return string One of: unknown, dev, stable, eom, eol
+     */
+    private function requestSymfonyState()
+    {
+        // get version information from cache or the roadmap
+        $versionCachePath = $this->kernel->getCacheDir().'/version_info.json';
+        if (file_exists($versionCachePath)) {
+            $versionInfo = json_decode(file_get_contents($versionCachePath), true);
+        } else {
+            $versionResponse = file_get_contents('http://symfony.com/roadmap.json?version='.$this->data['symfony_version']);
```

Comment:
Indeed:

 > An E_WARNING level error is generated if filename cannot be found, maxlength is less than zero, or if seeking to the specified offset in the stream fails.

Should I prefix it with `@`?